code/modules/projectiles/projectile/special/plasma.dm

```diff
@@ -28,13 +28,25 @@
 	mine_range = 5
 
 /obj/item/projectile/plasma/adv/mech
-	damage = 10
+	damage = 25
 	range = 9
+    var/pressure_decrease_active = FALSE
+    var/pressure_decrease = 0.2

```

Comment:
Also, the build is failing because you have spaces instead of tabs on these two lines. It's not visible in the webeditor, but _very_ visible in the raw file. If your browser refuses to outright input tabs, copy them from the original lines.

Reply:
Ah so that's why it's failing.

Reply:
That seems to have done the trick, thanks again.